components/bsx/Offer/OfferTable.vue

```diff
@@ -137,6 +152,35 @@ export default class OfferTable extends mixins(OfferMixin) {
   get urlPrefix() {
     return this.$store.getters.currentUrlPrefix
   }
+
+  get total(): number {
+    return this.offers.length
+  }
+
+  get showList(): any[] {
+    const endIndex = this.currentPage * this.itemsPerPage
+    return this.offers.slice(endIndex - this.itemsPerPage, endIndex)
+  }
+
+  public created() {
+    onApiConnect(async (api) => {
+      const currentBlock = await api.query.system.number()
+      this.currentBlock = currentBlock.toNumber()
+    })
+  }
+
+  public calcExpirationTime(expirationBlock: number) {
+    if (this.currentBlock === 0) {
+      return 'computing'
+    }
+    if (this.currentBlock > expirationBlock) {
+      return 'expired'
+    }
+    const secondsForEachBlock = 12
+    const diffSeconds =
+      secondsForEachBlock * (expirationBlock - this.currentBlock)
+    return formatSecondsToDuration(diffSeconds)
+  }
```

Comment:
pls delete `created()` and  `calcExpirationTime(expirationBlock: number)` function.
They are already in `OfferMixin`

Reply:
oh okay